salt/orch/kubernetes.sls

```diff
@@ -160,13 +160,13 @@ reboot_setup:
     - require:
       - salt: kube_master_setup
 
-dex_setup:
```

Comment:
Dex setup is already completed well before this stage, during the master node's highstate several steps up - hence replacing this with a "ensure its running".